react/Header/UserAccountMenu/UserAccountMenu.js

```diff
@@ -190,8 +183,8 @@ export default ({
           />
         ]
       })}
-    </Hidden>
-
+    </li>
+    <hr className={styles.menuSeparator} />
```

Comment:
I think the separator should only be rendered on desktop as the separation to "Company Reviews" looks odd to me. Does that sound right to you?
![Screen Shot 2019-03-13 at 2 11 48 pm](https://user-images.githubusercontent.com/912060/54250998-f557de00-4599-11e9-8789-e0deb430309e.png)

Reply:
Oops forgot to check the mobile menu. This change should definitely only be on desktop, cheers!
#mobilefirst4life